python/pyspark/ml/linalg/__init__.py

```diff
@@ -705,6 +705,23 @@ def __eq__(self, other):
             return Vectors._equals(self.indices, self.values, list(xrange(len(other))), other.array)
         return False
 
+    def __getattr__(self, item):
+        def wrapper(*args, **kwargs):
+            if _have_scipy:
+                csr = scipy.sparse.csr_matrix((np.append(self.values, 0),
```

Comment:
Super minor nit, but using named parameters might make this a bit easier when skimming the code (same for mllin)